app/views/layout.erb

```diff
@@ -1,10 +1,15 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-  <link rel="stylesheet" href="/application.css">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+  <link rel="stylesheet" href="/css/bootstrap.min.css">
+  <link rel="stylesheet" href="/css/application.css">
   <title>Skill Inventory</title>
 </head>
 <body>
-  <%= yield %>
+  <div class="container">
```

Comment:
when you have the yield in the container like this - every view will be rendered in a container. That is not necessarily a problem, but you need to be aware of that.